Makefile.common

```diff
@@ -55,7 +55,7 @@ ifneq ($(shell command -v gotestsum > /dev/null),)
 endif
 endif
 
-PROMU_VERSION ?= 0.14.0
+PROMU_VERSION ?= 0.15.0
```

Comment:
We don't update Makefile.common here. This happens in `prometheus/prometheus`.

Reply:
Need I revert code？